src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectSettingsPanel.java

```diff
@@ -52,7 +54,13 @@ public SonarLintProjectSettingsPanel(Project project) {
     tabs.insertTab("Bind to SonarQube project", null, rootBindPane, "Configure the binding of modules to a SonarQube server", 0);
     tabs.insertTab("File Exclusions", null, exclusionsPanel.getComponent(), "Configure which files to exclude from analysis", 1);
     tabs.insertTab("Analysis properties", null, rootPropertiesPane, "Configure analysis properties", 2);
+    tabs.insertTab("Analyzers Info", null, analyzersPanel.getPanel(), "Check which analyzers are currently loaded", 3);
```

Comment:
Maybe we could even drop the "Info", and name the tab just "Analyzers" ?

Reply:
That's fine too. @laurawacrenier  WDYT?

Reply:
I'm ok with both